label_studio/io_storages/base_models.py

```diff
@@ -255,8 +255,17 @@ def can_resolve_scheme(self, url: Union[str, None]) -> bool:
             return False
         # TODO: Search for occurrences inside string, e.g. for cases like "gs://bucket/file.pdf" or "<embed src='gs://bucket/file.pdf'/>"
         _, prefix = get_uri_via_regex(url, prefixes=(self.url_scheme,))
-        if prefix == self.url_scheme:
-            return True
+        bucket_uri = parse_bucket_uri(url, self)
+
+        # If there is a prefix and the bucket matches the storage's bucket/container/path
+        if prefix == self.url_scheme and bucket_uri:
+            # Check for different storage types
+            if hasattr(self, 'bucket') and bucket_uri.bucket == self.bucket:
+                return True
+            if hasattr(self, 'container') and bucket_uri.bucket == self.container:
```

Comment:
can you add comment - what storage does have 'container' ? the same about 'path'?